apps/authentication-service/cypress/support/commands.ts

```diff
@@ -39,29 +40,36 @@ import { JwtPayload } from "jsonwebtoken";
 //   }
 // }
 
-Cypress.Commands.add("loginClientAuth0", () => {
+Cypress.Commands.add("loginClientAuth0", (doUseRoleUser: boolean = true) => {
```

Comment:
This abstraction doesn't feel very extensible. I'd do something more like this: https://github.com/Universal-Connect-Project/ucp-hosted-app/blob/main/apps/ucp-ui/cypress/support/commands.ts